sdb/commands/exit.py

```diff
@@ -0,0 +1,31 @@
+#
+# Copyright 2019 Chuck Tuffli
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# pylint: disable=missing-docstring
+
+from typing import Iterable
+
+import drgn
+import sdb
+
+
+class Echo(sdb.Command):
```

Comment:
I realize this is too late, since this was just integrated, but I think the convention is for the class name to be related to the file name, i.e. `Exit` in this case.

Reply:
Good catch. New PR on its way.